crates/testlang/outputs/spec/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+# how to do the build.rs without an actual crate?
+[package]
+name = "testlang_spec"
```

Comment:
I understand the need for Rust/TypeScript outputs for `testlang`, but I wonder if we need the spec, given that it is a relatively heavy step, with lots of files to add/maintain manually over time.
Are there any consumers for this? if not, I suggest:

- removing `testlang/outputs/spec` crate.
- removing `testlang/inputs/language/docs` dir.

Reply:
On the other hand, because of this, I discovered #812 ...

Reply:
Given that we don't test the doc generation in any way, we should remove it.

Reply:
Sounds good.